x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -278,64 +293,73 @@ export class ExecuteReportTask implements ReportingTask {
 
             // Update job status to claimed
             report = await this._claimJob(task);
-
-            const { jobtype: jobType, attempts: attempt, max_attempts: maxAttempts } = task;
-            this.logger.info(
-              `Starting ${jobType} report ${jobId}: attempt ${attempt + 1} of ${maxAttempts}.`
-            );
-            this.logger.debug(`Reports running: ${this.reporting.countConcurrentReports()}.`);
           } catch (failedToClaim) {
             // error claiming report - log the error
             // could be version conflict, or no longer connected to ES
-            errorLogger(this.logger, `Error in claiming report!`, failedToClaim);
+            errorLogger(this.logger, `Error in claiming ${jobId}`, failedToClaim);
```

Comment:
When testing with a lot of jobs pending in the queue and a very low `xpack.reporting.queue.timeout` set, it was clear that the job ID needed to be added to many of log messages.